specs/darwin/connected_displays.table

```diff
@@ -0,0 +1,21 @@
+table_name("connected_displays")
+description("Provides information about the connected displays of the machine.")
+schema([
+    Column("name", TEXT, "The name of the display."),
+    Column("product_id", TEXT, "The product ID of the display."),
+    Column("serial_number", TEXT, "The serial number of the display."), 
+    Column("vendor_id", TEXT, "The vendor ID of the display."),
+    Column("display_week", INTEGER, "The manufacture week of the display. This field is 0 if not supported"), 
+    Column("display_year", INTEGER, "The manufacture year of the display. This field is 0 if not supported"),
+    Column("display_id", TEXT, "The display ID."),
+    Column("pixels", TEXT, "The number of pixels of the display."),
+    Column("resolution", TEXT, "The resolution of the display."),
+    Column("ambient_brightness", TEXT, "The ambient brightness of the display. This will be 1 if enabled and is 0 if disabled or not supported."),
```

Comment:
The column name and description imply it's the brightness level. But 0 or 1 is a boolean. So maybe `ambient_brightness_enabled` and change the description? Something else?